app/src/main/java/fr/free/nrw/commons/upload/depicts/DepictsFragment.java

```diff
@@ -87,6 +92,19 @@ public void onClick(View v) {
         addTextChangeListenerToSearchBox();
     }
 
+    /**
+     * Sets the Depicts subtitle
```

Comment:
Same here, could be more definitive. "Sets the Depict subtitle" is not actually what the method does. It decides the visibility of depics text view if multiplefile is not selecred as far as I can understand.